cypress/local-only/tests/integration/fileUpload/upload-court-issued-pdf-validation.cy.ts

```diff
@@ -0,0 +1,119 @@
+import { goToCase } from '../../../../helpers/caseDetail/go-to-case';
+
+import { MAX_FILE_SIZE_MB } from '../../../../../shared/src/business/entities/EntityConstants';
+import { attachFile } from '../../../../helpers/file/upload-file';
+import { loginAsDocketClerk1 } from '../../../../helpers/authentication/login-as-helpers';
+
+describe('upload court issued document validations', () => {
+  const docketNumber = '102-67'; // Any existing docket number works
+  const encoding = 'binary'; // Some Cypress tests were not mimicking real app behavior without manually setting the encoding to binary
```

Comment:
The reason (I think): corrupt/encrypted files make it difficult for cypress with [cypress-file-upload](https://www.npmjs.com/package/cypress-file-upload) to determine the right encoding. E.g., we want a corrupt pdf to be treated as a binary file, but because it is corrupt cypress thinks it was something else.